projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/read-case-flag-field.component.ts

```diff
@@ -103,4 +108,13 @@ export class ReadCaseFlagFieldComponent extends AbstractFieldReadComponent imple
 
     return null;
   }
+
+  public navigateBackToForm(fieldState: number): void {
+    this.router.navigate(['../createCaseFlagCaseFlagFormPage'], {
```

Comment:
The event can be on any path - needs to extracted from the start of the event - may need to put this in the router path data